vuepress/docs/next/tutorials/solution/install-standard-demo.md

```diff
@@ -30,11 +26,29 @@ There are numerous assets installed as part of the Standard Banking Demo. Entand
 
 ### Prerequisites
 
-- An Entando Application on any Kubernetes provider. Follow one of the [tutorials](../#operations) appropriate to your environment to install the Entando platform.
+- A working instance of Entando running on Kubernetes. [Install Entando on any Kubernetes provider](../#operations) or see [Getting Started](../../docs/getting-started/) for more information. 
 - The ent command line tool, installed and connected to your Kubernetes instance.
 
-### Installation Steps
-
+### Automatic Install via the Entando Hub
+Install the Standard Banking Demo by integrating the Entando Hub into your App Builder. 
+1. Log into your `App Builder` → `Repository` → `Select Registry` (kebob menu) 
```

Comment:
I think it's usually kebab rather than kebob. I don't feel strongly about it but since App Builder and Repository are not from the kebab menu it confuses me a little.